ci-operator/config/windup/windup-ui-tests/windup-windup-ui-tests-main__lp-interop-ocp4.12.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  dockerfile_path: dockerfiles/interop/Dockerfile
+  to: mtr-runner
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.12"
```

Comment:
We probably want to move straight to 4.13 for our first production runs. We've already moved our current interop testing to 4.13

Reply:
Resolved in [3fd91bd](https://github.com/openshift/release/pull/36181/commits/3fd91bd72531fc6e504ff9eb26657e0278dea6d1)